benchmarks/.gitignore

```diff
@@ -1,2 +1,3 @@
 data
-results
\ No newline at end of file
+results
+.venv
```

Comment:
the script makes a `venv` directory, not a `.venv` directory, should this be `venv`?

Reply:
Good catch! I changed it last second, but will update!